cernopendata/modules/fixtures/data/records/delphi-author-list-all.json

```diff
@@ -0,0 +1,4427 @@
+        },
+        {
+          "name": "Fayot, J.",
+          "affiliation": "LPNHE, IN2P3-CNRS, Univ.~Paris VI et VII, Tour 33 (RdC), 4 place Jussieu, FR-75252 Paris Cedex 05, France"
+        },
+        {
+          "name": "Fearnley, T.A.",
+          "affiliation": "Department of Physics, University of Bergen, Allegaten 55, NO-5007 Bergen, Norway"
+        },
+        {
+          "name": "Feindt, M.",
+          "affiliation": "CERN, CH-1211 Geneva 23, Switzerland; Institut fuer Experimentelle Kernphysik, Universitaet Karlsruhe, Postfach 6980, DE-76128 Karlsruhe, Germany"
+        },
+        {
+          "name": "Fenyuk, A.",
+          "affiliation": "Inst. for High Energy Physics, Serpukov P.O. Box 35, Protvino, (Moscow Region), Russian Federation"
+        },
+        {
+          "name": "Fernandez~Alonso, M.",
+          "affiliation": "Instituto de Fisica de Cantabria (CSIC-UC), Avda. los Castros s/n, ES-39006 Santander, Spain"
+        },
+        {
+          "name": "Fernandez, J.",
+          "affiliation": "Instituto de Fisica de Cantabria (CSIC-UC), Avda. los Castros s/n, ES-39006 Santander, Spain"
+        },
+        {
+          "name": "Ferrari, P.",
+          "affiliation": "Dipartimento di Fisica, Universita di Milano; INFN-MILANO, Via Celoria 16, IT-20133 Milan, Italy; Dipartimento di Fisica, Univ. di Milano-Bicocca; INFN-MILANO, Piazza delle Scienze 2, IT-20126 Milan, Italy"
+        },
+        {
+          "name": "Ferrer, A.",
+          "affiliation": "VALE IPNP of MFF, Charles Univ., Areal MFF, V Holesovickach 2, CZ-180 00, Praha 8, Czech RepublicIA"
+        },
+        {
+          "name": "Ferrer-Ribas, E.",
+          "affiliation": "Universite de Paris-Sud, Lab. de l'Accelerateur Lineaire, IN2P3-CNRS, Bat. 200, FR-91405 Orsay Cedex, France"
+        },
+        {
+          "name": "Ferro, F.",
+          "affiliation": "Dipartimento di Fisica, Universita di Genova; INFN, Via Dodecaneso 33, IT-16146 Genova, Italy"
+        },
+        {
+          "name": "Fichet, S.",
+          "affiliation": "LPNHE, IN2P3-CNRS, Univ.~Paris VI et VII, Tour 33 (RdC), 4 place Jussieu, FR-75252 Paris Cedex 05, France"
+        },
+        {
+          "name": "Filippas, T.A.",
+          "affiliation": "N Dipartimento di Fisica, Universit`a di Trieste; INFN, Via A. Valerio 2, IT-34127 Trieste, Italy; Istituto di Fisica, Universit`a di Udine, IT-33100 Udine, Italy- Physics Laboratory, University of Athens, Solonos Str. 104, GR-10680 Athens, Greece"
+        },
+        {
+          "name": "Firestone, A.",
+          "affiliation": "Department of Physics and Astronomy, Iowa State University, Ames IA 50011-3160, USA"
+        },
+        {
+          "name": "Fischer, P.-A.",
```

Comment:
`Fischer, P-A` or `Fischer P A` depending on the importance of hyphen. (here and elsewhere in the PR)